### DISCUSSION TITLE
Running vaultwarden behind gke ingress

### DISCUSSION THREAD

**Discussion by jona7o** [dani-garcia/vaultwarden#3274] [Q&A]
Hi, we have a problem with the websocket endpoint when we use gke and publish the notification hub path as a kubernetes ingress. The api implementation works fine because the Google Cloud Loadbalancer is able to request the _/_ path with **200 success**. Unfortunately the websocket endpoint can not be checked because the get response of _/notification/hub_ is not an **200 success**. Is there a solution how i can fix it or would it be possible to add a http get endpoint in the websocket service which always returns a **200 success** if the loadbalancer calls the specific path via http get? For example _/notification/hub/health _

**Comment by BlackDex** [comment]
Not really. And do you really need to check the health of that specific endpoint? In theory checking /alive should be more then enough for health checking. No other endpoints are needed.

The web-socket endpoint will not shutdown while the rest keeps working for example. Only way would be if you disable web-sockets and restart the application of course.

**Reply by jona7o** [reply]
The Problem is that the health check for each backend is mandatory. You cannot publish the given path to a separate backend port without a health check in the backend and the google loadbalancer always checks for 200 success, the only thing you can customize is the path to check.
